src/views/ModelPlan/CollaborationArea/Cards/ModelPlanCard/index.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Button,
+  Card,
+  CardBody,
+  CardFooter,
+  CardHeader
+} from '@trussworks/react-uswds';
+import {
+  GetModelPlanQuery,
+  TaskStatus,
+  useGetModelPlanQuery
+} from 'gql/gen/graphql';
+
+import UswdsReactLink from 'components/LinkWrapper';
+import Modal from 'components/Modal';
+import { Avatar } from 'components/shared/Avatar';
+import ShareExportModal from 'components/ShareExport';
+import { formatDateLocal } from 'utils/date';
+import {
+  getLatestModifiedDate,
+  ITSolutionsType,
+  StatusMessageType
+} from 'views/ModelPlan/TaskList';
+import { TaskListStatusTag } from 'views/ModelPlan/TaskList/_components/TaskListItem';
+
+import './index.scss';
+
+type GetModelPlanTypes = GetModelPlanQuery['modelPlan'];
+type OperationalNeedsType = GetModelPlanQuery['modelPlan']['operationalNeeds'][0];
+
+type ModelPlanCardType = {
+  modelID: string;
+  setStatusMessage: (message: StatusMessageType) => void;
+};
+
+const ModelPlanCard = ({ modelID, setStatusMessage }: ModelPlanCardType) => {
+  const { t: modelPlanCardT } = useTranslation('modelPlanCard');
+  const [isExportModalOpen, setIsExportModalOpen] = useState<boolean>(false);
+  const { data, loading } = useGetModelPlanQuery({
+    variables: {
+      id: modelID
+    }
+  });
+
+  const modelPlan = data?.modelPlan || ({} as GetModelPlanTypes);
+  const {
+    modifiedDts,
+    modifiedByUserAccount,
+    basics,
+    generalCharacteristics,
+    participantsAndProviders,
+    beneficiaries,
+    opsEvalAndLearning,
+    payments,
+    operationalNeeds = [],
+    taskListStatus
+  } = modelPlan;
+
+  const getITSolutionsStatus = (
+    operationalNeedsArray: OperationalNeedsType[]
+  ) => {
+    const inProgress = operationalNeedsArray.find(need => need.modifiedDts);
+    return inProgress ? TaskStatus.IN_PROGRESS : TaskStatus.READY;
+  };
```

Comment:
We have a field on the model plan for this use `opSolutionLastModifiedDts`. Can you that calculated field instead of this util.  I think what was on the task list was legacy and can probably be converted as well

Reply:
I initially just copied this over from task list to determine the TaskStatus. However, I looked at the snippet again and realized that it is calling for some things I actually do not need, like modifiedDate. 

So I just removed that piece and kept the util function to determine the status